plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -100,15 +100,15 @@ vehicle. These parameters are critical to the tuning process.
 - :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>`: 80Hz for 5 inch props, 40Hz for 10 inch props, 20Hz for 20 inch props
 - :ref:`Q_A_ACCEL_P_MAX <Q_A_ACCEL_P_MAX>`: 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
 - :ref:`Q_A_ACCEL_R_MAX <Q_A_ACCEL_R_MAX>`: 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
-- :ref:`Q_A_ACCEL_Y_MAX <Q_A_ACCEL_Y_MAX>`: 27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props
+- :ref:`Q_A_ACCEL_Y_MAX <Q_A_ACCEL_Y_MAX>`: 6000 for 6 inch props,27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props
```

Comment:
did you mean 60000 ?

Reply:
no, both my and @andyp1per small vtols have this value, but whatever you guys think...I know I had one that did not stop swaying until it got lowered to this value although I did not experiment with intermediate values...just used Andy's value for his vtols

Reply:
changed to 1/4 multi values per Pete H///also updated curve labels